openshift/template.yaml

```
@@ -92,6 +97,11 @@ objects:
               secretKeyRef:
                 name: aws-sqs
                 key: aws_secret_access_key
+          - name: ENCRYPTION_KEY_FOR_SNYK_TOKEN
+            valueFrom:
+              secretKeyRef:
+                name: snyk-encryption
+                key: encryption_key_for_snyk_token
```

Short, Precise, Easy to read
```suggestion
                key: snyk_encryption_token
```

---

It is an encryption key for the API token.

---

ok, Can we summarize it in 3 letter variable name..?

---

Is this a standard that we need to follow?

---

3 letter looks more readable..! If we promote readiabiility, then it can be good standard to follow..

---

Isn't readability about clearly expressing what the variable means?  The only name that I can think of is encryption_key_snyk_token.

---

Fine. Settling with `encryption_key_for_snyk_token` then